app/javascript/local/suggestions.js

```diff
@@ -16,11 +16,6 @@ Input.prototype.init = function () {
   if (suggestionsSourceId) {
     this.suggestions = document.getElementById(suggestionsSourceId)
 
-    this.$formGroup.setAttribute('role', 'combobox')
-    this.$formGroup.setAttribute('aria-owns', this.$module.getAttribute('id'))
```

Comment:
I removed aria-owns attribute as according to the [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-owns), it doesn't need to set if the element is a child of the parent and in this case the input element is a child of the form container

Reply:
Oh interesting. It says:
> In older ARIA specs, this was aria-owns rather than aria-controls, so you may see aria-owns in older combobox implementations. The aria-owns in the code should be updated to aria-controls.

So yeah, I think removing the aria owns and leaving the aria-controls is correct in this case 👍

Reply:
Ah perfect 👍